api/client/builder/types.go

```diff
@@ -1307,6 +1277,209 @@ func (p *ExecutionPayloadDeneb) ToProto() (*v1.ExecutionPayloadDeneb, error) {
+	}
+	executionRequests, err := bb.ExecutionRequests.ToProto()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to convert ExecutionRequests")
+	}
+	return &eth.BuilderBidElectra{
+		Header:             header,
+		BlobKzgCommitments: kzgCommitments,
+		ExecutionRequests:  executionRequests,
+		// Note that SSZBytes() reverses byte order for the little-endian representation.
+		// Uint256.Bytes() is big-endian, SSZBytes takes this value and reverses it.
+		Value:  bytesutil.SafeCopyBytes(bb.Value.SSZBytes()),
+		Pubkey: bytesutil.SafeCopyBytes(bb.Pubkey),
+	}, nil
+}
+
+// ExecutionRequestsV1 is a wrapper for different execution requests
+type ExecutionRequestsV1 struct {
+	Deposits       []*DepositRequestV1       `json:"deposits"`
+	Withdrawals    []*WithdrawalRequestV1    `json:"withdrawals"`
+	Consolidations []*ConsolidationRequestV1 `json:"consolidations"`
+}
+
+func (er *ExecutionRequestsV1) ToProto() (*v1.ExecutionRequests, error) {
+	if uint64(len(er.Deposits)) > params.BeaconConfig().MaxDepositRequestsPerPayload {
+		return nil, fmt.Errorf("deposit requests count %d exceeds the maximum %d", len(er.Deposits), params.BeaconConfig().MaxDepositRequestsPerPayload)
+	}
+	deposits := make([]*v1.DepositRequest, len(er.Deposits))
+	for i, dep := range er.Deposits {
+		d, err := dep.ToProto()
+		if err != nil {
+			return nil, err
+		}
+		deposits[i] = d
+	}
+	if uint64(len(er.Withdrawals)) > params.BeaconConfig().MaxWithdrawalRequestsPerPayload {
+		return nil, fmt.Errorf("withdrawal requests %d exceeds the maximum %d", len(er.Withdrawals), params.BeaconConfig().MaxWithdrawalRequestsPerPayload)
+	}
+	withdrawals := make([]*v1.WithdrawalRequest, len(er.Withdrawals))
+	for i, wr := range er.Withdrawals {
+		w, err := wr.ToProto()
+		if err != nil {
+			return nil, err
+		}
+		withdrawals[i] = w
+	}
+	if uint64(len(er.Consolidations)) > params.BeaconConfig().MaxConsolidationsRequestsPerPayload {
+		return nil, fmt.Errorf("consolidation requests %d exceeds the maximum %d", len(er.Consolidations), params.BeaconConfig().MaxConsolidationsRequestsPerPayload)
+	}
+	consolidations := make([]*v1.ConsolidationRequest, len(er.Consolidations))
+	for i, con := range er.Consolidations {
+		c, err := con.ToProto()
+		if err != nil {
+			return nil, err
+		}
+		consolidations[i] = c
+	}
+	return &v1.ExecutionRequests{
+		Deposits:       deposits,
+		Withdrawals:    withdrawals,
+		Consolidations: consolidations,
+	}, nil
+}
+
+// BuilderBidElectra is a field of ExecHeaderResponseElectra.
+type BuilderBidElectra struct {
+	Header             *ExecutionPayloadHeaderDeneb `json:"header"`
+	BlobKzgCommitments []hexutil.Bytes              `json:"blob_kzg_commitments"`
+	ExecutionRequests  *ExecutionRequestsV1         `json:"execution_requests"`
+	Value              Uint256                      `json:"value"`
+	Pubkey             hexutil.Bytes                `json:"pubkey"`
+}
+
+type ExecutionPayloadHeaderElectra = ExecutionPayloadHeaderDeneb
```

Comment:
You removed `ExecutionPayloadElectra` but not this type (see comment https://github.com/prysmaticlabs/prysm/pull/14344#discussion_r1918953189)